code/src/index.css

```diff
@@ -1,13 +1,174 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;400&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: "Roboto", sans-serif;
+  background-color: rgb(228, 169, 165);
+}
+
+.question-card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid black;
+  box-shadow: 5px 10px;
+  width: 500px;
+  margin: 20px auto 0 auto;
+  background-color: rgb(237, 237, 237);
+}
+
+.question-container {
+  margin-left: 20px;
+  margin-top: 20px;
+}
+
+.question {
+  font-weight: 300;
+  font-size: 24px;
+  padding-bottom: 10px;
+}
+
+.submit-btn {
+  width: 250px;
+  border-radius: 30px;
+  border: none;
+  background-color: rgb(228, 169, 165);
+  padding: 10px 0;
+  margin: 20px;
+}
+
+.icon-btn {
+  color: red;
+  padding: 0 10px;
+  font-size: 18px;
+}
+
+textarea {
+  margin-top: 20px;
+  padding: 10px;
+  padding-bottom: 40px;
+  width: 90%;
+  background-color: white;
+  resize: none;
+}
+
+.answer-card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid black;
+  width: 500px;
+  margin: 20px auto 0 auto;
+  background-color: white;
+
+  box-shadow: 5px 10px;
+}
+
+.answer {
+  margin: 0 20px;
+  word-wrap: break-word;
+  white-space: pre-wrap;
+}
+
+.heart-btn {
+  background-color: white;
+  border: none;
+  color: grey;
+}
+
+.heart {
+  width: 50px;
+  color: red;
+  background-color: rgb(237, 237, 237);
+  border-radius: 50%;
+  padding: 10px;
+  font-size: 20px;
+}
+
+.heart-liked {
+  width: 50px;
+  color: red;
+  background-color: rgb(228, 169, 165);
+  border-radius: 50%;
+  padding: 10px;
+  font-size: 20px;
+}
+.counter {
+  background-color: white;
+  border: 1px solid black;
+  width: 40px;
+  margin-left: 20px;
+  padding: 2px 10px;
+  text-align: center;
+}
+
+.invalid {
+  color: rgb(223, 94, 85);
+  margin-left: 25px;
+  padding-right: 2px;
+}
+
+.valid {
+  color: green;
+  margin-left: 20px;
+}
+
+.invalid-input {
```

Comment:
Good work with the validation, and I nice touch that the input window changes color as an extra signal.